src/components/molecules/HamDropDown/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+import Styled from 'styled-components';
+import arrowDown from 'assets/images/ham_arrowDown.svg';
+import arrowUp from 'assets/images/ham_arrowUp.svg';
+import Button from '../../atoms/Button/index';
+
+export interface IProps {
+  className?: string;
+  title: string;
+  list: string[];
+}
+
+function HamDropDown({ ...props }: IProps): React.ReactElement {
+  const { title, list } = props;
```

Comment:
```suggestion
function HamDropDown({ title, list }: IProps): React.ReactElement {
```
우리 이렇게 사용하도록 한 것 아니었나용?